.test-cover.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
Can you add a brief description describing what this script is intending to do? I know these type of scripts are typically less readable than normal given that they're just working around Travis/Coveralls quirks, but it'd be nice to know why we need this.